### ISSUE TITLE
no key signature

### ISSUE THREAD

**Issue by jefftrevino** [siennamw/mei-tools-atom#4]
The Chopin demo file renders fine in Atom for me, but it omits the key signature.

**Comment by siennamw** [comment]
Chopin Etude op. 10 no. 9 from the MEI examples?  Here is what I see:
![chopin](https://user-images.githubusercontent.com/15326291/43049271-0aeee966-8db2-11e8-9225-6c2124543ed8.png)
Do we have different versions of this file, perhaps?

**Comment by jefftrevino** [comment]
The version I'm using is from the link provided on the atom package index page, in the readme. I paste its contents into an editor and save it as an .mei file. The file displays without error with the package, but it does not include a key signature: 
![screen shot 2018-07-22 at 2 09 28 pm](https://user-images.githubusercontent.com/716685/43050126-ea4c6074-8db8-11e8-84a2-19b16e5cf347.png)

**Comment by siennamw** [comment]
Yes, slightly different versions of the same file.  My guess is that this will be solved by updating the version of _Verovio_ that we're using.
